packages/fxa-content-server/server/lib/routes/get-index.js

```diff
@@ -52,6 +52,9 @@ module.exports = function (config) {
   const FEATURE_FLAGS_FXA_STATUS_ON_SETTINGS = config.get(
     'featureFlags.sendFxAStatusOnSettings'
   );
+  const FEATURE_FLAGS_RESET_PWD_WITH_CODE = config.get(
+    'featureFlags.resetPasswordWithCode'
+  );
```

Comment:
I don't think we need this since we are not implementing the feature in Backbone.   Or are we?

Reply:
We are not implementing in backbone 🙅‍♀️ I was mislead by the line above that adds another flag that seems to only be used for react. Removing the reset password one from here, I've checked that it's not needed for fxa-settings. Thanks!